src/System.Globalization/tests/System.Globalization.Tests.csproj

```diff
@@ -11,10 +11,23 @@
     <IncludePerformanceTests>true</IncludePerformanceTests>
   </PropertyGroup>
   <!-- Default configurations to help VS understand the configurations -->
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net462_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net462_Release|AnyCPU'" />
+
+  <PropertyGroup>
+    <IsWindowsDesktop Condition="'$(IsWindowsDesktop)' =='' And '$(TargetGroup)' == 'net45'">true</IsWindowsDesktop>
```

Comment:
We should use the TargetGroup name based on the convention. So the define should be `net45`.